pkg/dataplane/http/context.go

```diff
@@ -1221,10 +1221,26 @@ func (c *context) encodeTypedAttributes(attributes map[string]interface{}) (map[
 			return nil, fmt.Errorf("unexpected attribute type for %s: %T", attributeName, value)
 		case int:
 			typedAttributes[attributeName]["N"] = strconv.Itoa(value)
+		case uint:
+			typedAttributes[attributeName]["N"] = strconv.FormatUint(uint64(value), 10)
+		case int8:
+			typedAttributes[attributeName]["N"] = strconv.Itoa(int(value))
+		case uint8:
+			typedAttributes[attributeName]["N"] = strconv.Itoa(int(value))
+		case int16:
+			typedAttributes[attributeName]["N"] = strconv.Itoa(int(value))
+		case uint16:
+			typedAttributes[attributeName]["N"] = strconv.Itoa(int(value))
+		case int32:
+			typedAttributes[attributeName]["N"] = strconv.FormatInt(int64(value), 10)
+		case uint32:
+			typedAttributes[attributeName]["N"] = strconv.FormatUint(uint64(value), 10)
 		case uint64:
 			typedAttributes[attributeName]["N"] = strconv.FormatUint(value, 10)
 		case int64:
 			typedAttributes[attributeName]["N"] = strconv.FormatInt(value, 10)
+		case float32:
+			typedAttributes[attributeName]["N"] = strconv.FormatFloat(float64(value), 'E', -1, 64)
```

Comment:
I guess this can be simplified to
```suggestion
			typedAttributes[attributeName]["N"] = strconv.FormatFloat(value, 'E', -1, 32)
```
?

Reply:
FormatFlow expects float64 so you cannot pass value without conversion. Also internaly we work only with 64 bit, so I would leave last argument 64 as well

Reply:
Right, the conversion to `float64` needs to be there. From https://pkg.go.dev/strconv#FormatFloat, it seems like the last parameter needs to be set to 32 for correctness, or it will introduce arbitrary digits at the end.

Reply:
I prefer arbitrary digits to keep the format the same across all the floats